frontends/concrete-python/tests/execution/test_matmul.py

```diff
@@ -122,8 +127,12 @@ def test_matmul(lhs_shape, rhs_shape, bounds, helpers):
 
     minimum, maximum = bounds
 
-    lhs_cst = list(np.random.randint(minimum, maximum, size=lhs_shape))
-    rhs_cst = list(np.random.randint(minimum, maximum, size=rhs_shape))
+    def rgen(shape):
+        """Generate a random matrix with values within minimum and maximum."""
+        return np.random.randint(minimum, maximum, size=shape)
+
+    lhs_cst = list(rgen(lhs_shape))
+    rhs_cst = list(rgen(rhs_shape))
```

Comment:
Please use `np.random.randint` with minimum, maximum and size please. `rgen` captures minimum and maximum and it's not easy to grasp what the values will contain just from the usage.

Reply:
(example below)